dev-tools/omdb/tests/successes.out

```diff
@@ -71,45 +71,6 @@ loading inventory collections ... done.
 loading blueprints ... done.
 wrote <REDACTED>
 =============================================
-EXECUTING COMMAND: omdb ["db", "services", "list-instances"]
```

Comment:
I think this is covered by our `blueprint.display()` output. I added a test for `nexus blueprint show <ID>` in [fcf547a](https://github.com/oxidecomputer/omicron/pull/5287/commits/fcf547adbacf1a644df5cab4ed478a5a28f243a7), then realized we were still using debug instead of `.display()` so tweaked that in [3fd0288](https://github.com/oxidecomputer/omicron/pull/5287/commits/3fd02881ba2fb47e4ebb804dc3cf8bda5fd03922).